plutus-playground/deployment/nixops/default-machine.nix

```diff
@@ -0,0 +1,40 @@
+{ machines, stdOverlays, ... }: node: pkgs:
+{
+      nixpkgs.overlays = stdOverlays;
+      nix = {
+        nixPath = [ "nixpkgs=https://github.com/NixOS/nixpkgs-channels/archive/nixos-18.09.tar.gz"
```

Comment:
Here's the other suspicious nixpkgs.

Reply:
Sure looks like this file gets eventually imported into other stuff.

Reply:
hmm, are you sure we should be using the pinned nixpkgs here rather than the official nixos one, this is for the machine, not for the things we are building to go on it. This means we keep the machine up to date with nixos rather than having to wait for updates to the IOHK nixpkgs

Reply:
Hm, I guess you're right, there's nothing that stops us *running* our stuff on a machine where the services come from a different nixpkgs. Fair point - but can you add a comment for posterity.